manifests/charts/gateways/istio-egress/templates/_affinity.tpl

```diff
@@ -11,6 +11,7 @@ nodeAffinity:
 {{- define "nodeAffinityRequiredDuringScheduling" }}
       nodeSelectorTerms:
       - matchExpressions:
+{{- if .global.multiArchs }}
```

Comment:
maybe we can make it `if .global.arch`? Then its just one existing flag. If its empty, we do nothing. Otherwise we add it?

Reply:
I thought about it but does that mean we are deleting the archs from the `.global.arch`? I am not sure if we still need them for the preferred archs in lines https://github.com/istio/istio/blob/master/manifests/charts/gateways/istio-egress/templates/_affinity.tpl#L31-L43? If we are okay with deleting the archs, I can do it that way.

Reply:
I think we should. The current list doesn'r even make sense, Istio only works on amd64 currently.

Reply:
then do you want me to delete them altogether or leave only amd64 in there? that would solve the multh-arch issue while keeping the one it actually works.